app/archivesspace/archivesspace.controller.js

```diff
@@ -534,6 +555,7 @@ controller('ArchivesSpaceEditController', ['$uibModalInstance', 'levels', 'level
   vm.end_date = end_date;
   vm.date_expression = date_expression;
   vm.note = note;
+  vm.accessrestrict_note = note;
```

Comment:
`accessrestrict` is the underlying name of this category of note in ArchivesSpace.